v22.2/cluster-setup-troubleshooting.md

```diff
@@ -304,6 +304,30 @@ The solution is to add connection retry logic to your application.
 
 ## Clock sync issues
 
+#### A node's timezone data has changed
```

Comment:
Just noticing that these entries are h4, while the parent section is an h2. This may be intended, but it caught my eye.

Reply:
Yeah, I didn't want to deal with that in this PR. I suspect it might have been to suppress them in the in-page TOC.